circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
```

Comment:
This would break the cache (if it worked with 1.9, we didn't get it to work, right @satra?). Caching does not work with 1.10.